weekly-scan/scheduler.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: weekly-scan-scheduler-template
+labels:
+  type: "scheduler"
+  app: "weekly-scan"
+objects:
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: weekly-scan-scheduler-script
+    data:
+      script: |-
+        OS_TOKEN=`cat /tmp/jenkins-secret/token`
+        oc login ${OS_MASTER} --insecure-skip-tls-verify=true --token=${OS_TOKEN};
+        if [[ `oc get bc -o name --selector app=weekly-scan --selector type=pipeline | wc -l` == 0 ]]; then
+            echo "No projects to trigger";
+            exit 0
+        fi
+        for item in `oc get bc -o name --selector app=weekly-scan --selector type=pipeline`; do
+            oc start-build ${item};
+            sleep ${BATCH_DELAY}
+        done
+  - apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      name: weekly-scan-scheduler
+    spec:
+      schedule: "*/1 * * * *"
+      jobTemplate:
+        spec:
+          template:
+            metadata:
+            spec:
+              containers:
+                - name: weekly-scan-scheduler
+                  image: ${JENKINS_SLAVE_IMAGE_NAME}
+                  command: ["/bin/sh",  "/tmp/entry/script"]
```

Comment:
Yes it is tested. Infact, entrypoint will fail here.

This is because the entrypoint script handles some permission stuff to run this container on Openshift